aiinfra-cluster/modules/aiinfra-network/outputs.tf

```diff
@@ -19,6 +19,16 @@ output "network_name" {
   value       = local.primary_network.network_name
 }
 
+output "network_id" {
```

Comment:
network_id and subnet_address might not be necessary

Reply:
Similar to above, removed.